### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by galizien**
Lesson 1

### CODE REVIEW

File: lesson1/assets/dots.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<svg width="227" height="226" viewBox="0 0 227 226" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Maxim-Bondar) [inline lesson1/assets/dots.svg:1]: svg must be minified by https://jakearchibald.github.io/svgomg/

File: lesson1/index.html [hunk 2]
```diff
@@ -0,0 +1,934 @@
...
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick-theme.min.css"
+          integrity="sha512-17EgCFERpgZKcm0j0fEq1YCJuyAWdz9KUtv1EjVuaOz8pDnh/0nZxmU6BBXwaaxqoi9PQXnRWqlcDB027hgv9A=="
+          crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+     <link rel="stylesheet" href="css/main.css">
+     <title>Document</title>
+</head>
+
+<body>
+     <!-- <div class="wholse-content"> -->
```
Comment (Maxim-Bondar) [inline lesson1/index.html:24]: remove comment

File: lesson1/index.html [hunk 3]
```diff
@@ -0,0 +1,934 @@
...
+     <header>
+          <div class="hamburger-menu">
+               <img src="assets/logo-nav.png" class="logo" alt="">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:27]: attribute `alt` shouldn't be empty

File: lesson1/index.html [hunk 4]
```diff
@@ -0,0 +1,934 @@
...
+               <input id="menu__toggle" type="checkbox" />
```
Comment (Maxim-Bondar) [inline lesson1/index.html:28]: css implenetation is not accectable for this task, hamburger menu should be implemented by js

File: lesson1/index.html [hunk 5]
```diff
@@ -0,0 +1,934 @@
...
+                         <li><a class="menu-item" href="#">Partners</a></li>
+                         <li><a class="menu-item" href="#">Company</a></li>
+                         <li><a class="menu-item" href="#">Tips & Tools</a></li>
+                         <li><a class="menu-item" href="#">Blog</a></li>
+                    </ul>
+                    <div class="buttons">
+                         <button class="button-first">Get Skout</button>
+                         <button class="button-second">Log In</button>
+                    </div>
+
```
Comment (Maxim-Bondar) [inline lesson1/index.html:45]: use `.prittier` for code formating

File: lesson1/index.html [hunk 6]
```diff
@@ -0,0 +1,934 @@
...
+               </div>
+          </div>
+
+          <div class="header-laptop">
+               <img src="assets/logo-nav.png" class="laptop-logo" alt="">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:50]: logo should be a link

File: lesson1/index.html [hunk 7]
```diff
@@ -0,0 +1,934 @@
...
+               <div>
+                    <div class="header-links">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:52]: it should be a navigation list instead of div

File: lesson1/index.html [hunk 8]
```diff
@@ -0,0 +1,934 @@
...
+                         <a class="laptop-item" href="#">Why Skout</a>
+                         <a class="laptop-item" href="#">Products</a>
+                         <a class="laptop-item" href="#">Partners</a>
+                         <a class="laptop-item" href="#">Company</a>
+                         <a class="laptop-item" href="#">Tips & Tools</a>
+                         <a class="laptop-item" href="#">Blog</a>
+                    </div>
+               </div>
+               <div class="header-buttons">
+                    <button class="header-button-first">Get Skout</button>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:62]: these buttons has the same style , you don't need classes for each button

File: lesson1/index.html [hunk 9]
```diff
@@ -0,0 +1,934 @@
...
+                    <button class="header-button-second">Log In</button>
+               </div>
+          </div>
+
+     </header>
+
+     <section class="hero">
+          <div class="title">Tips & Tools</div>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:70]: use title tags instead of div `h1 .... h6`

File: lesson1/index.html [hunk 10]
```diff
@@ -0,0 +1,934 @@
...
+     </section>
+
+
+     <div class="tabs">
+          <div class="container">
+               <a href="#tips" class="tab">Tips</a>
+               <div class="vl"></div>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:77]: uselles element , it can be implemented by pseudo elements

File: lesson1/index.html [hunk 11]
```diff
@@ -0,0 +1,934 @@
...
+               <a href="#glossary" class="tab">Glossary</a>
+               <div class="vl"></div>
+               <a href="#faq" class="tab">FAQs</a>
+          </div>
+     </div>
+
+     <section class="content">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:84]: here better to use tag `main`

File: lesson1/index.html [hunk 12]
```diff
@@ -0,0 +1,934 @@
...
+          <div class="tips-container">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:85]: redundant wrapper

File: lesson1/index.html [hunk 13]
```diff
@@ -0,0 +1,934 @@
...
+               <div class="tips-content" id="tips">
+                    <div class="line"></div>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:87]: useless element, it shoulb be implemented by css

File: lesson1/index.html [hunk 14]
```diff
@@ -0,0 +1,934 @@
...
+                    <div class="did">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:88]: class `.did` is not obvious, difficult to understand what this class do

File: lesson1/index.html [hunk 15]
```diff
@@ -0,0 +1,934 @@
...
+                         <img src="assets/skout-tips.svg" alt="skout-tips" class="content-image">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:89]: it should be a text instead of image

File: lesson1/index.html [hunk 16]
```diff
@@ -0,0 +1,934 @@
...
+                    </div>
+
+                    <div class="count-container">
+                         <div class="count-line">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:93]: uselles element it should be made by css

File: lesson1/index.html [hunk 17]
```diff
@@ -0,0 +1,934 @@
...
+                              <div class="c-line"></div>
+                         </div>
+
+                         <div class="accordain-block">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:97]: typo in word accordain

File: lesson1/index.html [hunk 18]
```diff
@@ -0,0 +1,934 @@
...
+                              <a class="accordion">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:98]: it semantically not correct to put `div` inside `link`, by the way here you should use `button` with `span` inside

File: lesson1/index.html [hunk 19]
```diff
@@ -0,0 +1,934 @@
...
+                                   <div class="number">1</div>
+                                   <div class="text">System Patching</div>
+                                   <div class="plus"><img src="assets/plus.png" alt=""></div>
+                                   <div class="plus-laptop"><img src="assets/plus-laptop.png" height="35px" width="35px"
+                                             alt=""></div>
+                              </a>
+                              <div class="panel">
+                                   <div class="laptop-panels">
+                                        <div class="laptop-first-panel">
+                                             <div class="laptop-panel-title">System Patching</div>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:108]: use appropriate tags for title istead of div

File: lesson1/index.html [hunk 20]
```diff
@@ -0,0 +1,934 @@
...
+                         <div class="accordain-block">
+                              <a class="accordion">
+                                   <div class="number">1</div>
+                                   <div class="text">System Patching</div>
+                                   <div class="plus"><img src="assets/plus.png" alt=""></div>
+                                   <div class="plus-laptop"><img src="assets/plus-laptop.png" height="35px" width="35px"
+                                             alt=""></div>
+                              </a>
+                              <div class="panel">
+                                   <div class="laptop-panels">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:106]: a lot of useless wrappers, please check it can be simplify

File: lesson1/index.html [hunk 21]
```diff
@@ -0,0 +1,934 @@
...
+                                        <div class="laptop-first-panel">
+                                             <div class="laptop-panel-title">System Patching</div>
+                                             <div class="laptop-panel-picture">
+                                                  <img src="assets/circle.png" width="370px" height="370px" alt="">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:110]: image shouldn't be hardcoded inline, it must inheirite parent parent wrapper size

File: lesson1/index.html [hunk 22]
```diff
@@ -0,0 +1,934 @@
...
+                    </div>
+
+                    <div class="did">
+                         <img src="assets/skout-glossary.svg" alt="skout-tips" class="content-image">
+                    </div>
+                    <div class="background">
+                         <div class="slider-container">
+                              <div class="slider-titles">
+                                   <div>
+                                        <div class="t">IP Address</div>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:402]: what class `t` means, we have discussed class names on the last meeting, the class name must be obviously for everyone

File: lesson1/index.html [hunk 23]
```diff
@@ -0,0 +1,934 @@
...
+                         <div class="line"></div>
+                    </div>
+
+                    <div class="did">
+                         <img src="assets/skout-glossary.svg" alt="skout-tips" class="content-image">
+                    </div>
+                    <div class="background">
+                         <div class="slider-container">
+                              <div class="slider-titles">
+                                   <div>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:401]: it should be a list

File: lesson1/index.html [hunk 24]
```diff
@@ -0,0 +1,934 @@
...
+                                   <div class="laptop-slider-t">Ransomware</div>
+                                   <div class="laptop-slider-t">Session Hijacking</div>
+                                   <div class="laptop-slider-t">Social Engineering</div>
+                              </div>
+
+                         </div>
+                         <div>
+                              <div class="single-item">
+                                   <div class="slider-title-text">
+                                        <div class="slider-title">IP Address</div>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:431]: it should be a heading tag

File: lesson1/index.html [hunk 25]
```diff
@@ -0,0 +1,934 @@
...
+                                        <div class="slider-text">Malicious software that blocks access to a system until
```
Comment (Maxim-Bondar) [inline lesson1/index.html:432]: it should be a paragraph tag

File: lesson1/index.html [hunk 26]
```diff
@@ -0,0 +1,934 @@
...
+
+               </div>
+          </div>
+
+
+
+
+          </div>
+
+          <div class="faq-container">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:601]: for this section the same comments as for accordion above

File: lesson1/index.html [hunk 27]
```diff
@@ -0,0 +1,934 @@
...
+                    </div>
+               </div>
+
+          </div>
+
+
+
+     </section>
+     <footer class="mobile">
+          <div class="footer-links">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:801]: should be a list with links inside

File: lesson1/index.html [hunk 28]
```diff
@@ -0,0 +1,934 @@
...
+               <div class="footer-link">Blog</div>
+               <div class="footer-link">Company</div>
+               <div class="footer-link">Terms & Conditions</div>
+               <div class="footer-link">Privacy Policy</div>
+               <div class="footer-link">Careers</div>
+               <div class="footer-link">Call the SOC</div>
+               <div class="phone-number">1-855-838-4500</div>
+          </div>
+          <div class="for-line">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:810]: useless element

File: lesson1/index.html [hunk 29]
```diff
@@ -0,0 +1,934 @@
...
+               <div class="line"></div>
+          </div>
+          <div class="address">
+               <div class="phone-number">New York</div>
+               <div class="phone-number">+1 631-203-6600</div>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:815]: should be a link with `href="tel:...."`

File: lesson1/index.html [hunk 30]
```diff
@@ -0,0 +1,934 @@
...
+               <div class="address-street">423 W 55th St. 7th Floor
+                    New York, NY, 10019</div>
+               <a href="#" class="view-map">View Map</a>
+
+          </div>
+          <div class="for-line">
+               <div class="last-line"></div>
+          </div>
+          <div class="socials">
+               <div class="socials-links">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:845]: should be a list

File: lesson1/index.html [hunk 31]
```diff
@@ -0,0 +1,934 @@
...
+               </div>
+               <div class="for-footer-logo">
+                    <img src="assets/output-onlinepngtools (7).png" alt="cybersecurity" width="105px" height="50px">
+               </div>
+          </div>
+
+     </footer>
+
+     <!-- LAPTOP FOOTER -->
+     <footer class="laptop">
```
Comment (Maxim-Bondar) [inline lesson1/index.html:859]: you don't need a tons of similar sections, remove useless footer and make changes by css

File: lesson1/index.html [hunk 32]
```diff
@@ -0,0 +1,934 @@
...
+          </div>
+     </footer>
+     </div>
+     <script src="https://code.jquery.com/jquery-3.6.0.min.js"
+          integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+     <script src="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick.js"
+          integrity="sha512-WNZwVebQjhSxEzwbettGuQgWxbpYdoLf7mH+25A7sfQbbxKeS5SQ9QBf97zOY4nOlwtksgDA/czSTmfj4DUEiQ=="
+          crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+     <!-- <script src="script/stick.js"></script>
```
Comment (Maxim-Bondar) [inline lesson1/index.html:926]: remove commented code

**Review by Maxim-Bondar — CHANGES_REQUESTED** [review]
- remove node_modules from request
- you don't have `scss` styles in your request
- remove all redundant sections, which you created for each section where you must make styles changes by media queries
- hero section must have full width https://prnt.sc/13llt2v
- don't have an underline for selected elements https://prnt.sc/13llu59
- buttons don't have a hover state as was mentioned in technical task https://prnt.sc/13llvzh
- not as is the design https://prnt.sc/13llxlz 
- slider is broken https://prnt.sc/13llylj
- arrow should aligned right https://prnt.sc/13llzun
- footer should have a full width https://prnt.sc/13lm0i1
- broken sections on mobile https://prnt.sc/13lm39a  https://prnt.sc/13lm43o https://prnt.sc/13lm53b

File: lesson1/assets/dots.svg (hunk 1 above)
Comment (galizien) [reply lesson1/assets/dots.svg:1]: done

File: lesson1/index.html (hunk 2 above)
Comment (galizien) [reply lesson1/index.html:24]: removed comment

**Pull request closed without merging**